apps/api/v2/src/modules/oauth-clients/services/oauth-clients-users.service.ts

```diff
@@ -36,17 +36,15 @@ export class OAuthClientUsersService {
       const email = this.getOAuthUserEmail(oAuthClientId, body.email);
       user = (
         await createNewUsersConnectToOrgIfExists({
-          usernamesOrEmails: [email],
-          input: {
-            teamId: organizationId,
-            role: "MEMBER",
-            usernameOrEmail: [email],
-            isOrg: true,
-            language: "en",
-          },
+          invitations: [{
+            usernameOrEmail: email,
+            role: "MEMBER"
+          }],
+          teamId: organizationId,
```

Comment:
cc @ThyMinimalDev Changed the argument. `input` was too generic. So, removed that as we didn't need everything from it.

`invitations` instead of usernameOrEmails now which support role per user.

Reply:
platform-libraries are versioned, we can't change code impacting functions from platform-libraries without publishing new version of our package

Reply:
ill make a pr